docs/conf.py

```diff
@@ -301,3 +301,7 @@ def do_not_plot(*args, **kwargs):
     # Many links for PRs from our release notes. Slow and unlikely to cause issues.
     'https://github.com/scipp/scipp/pull/[0-9]+',
 ]
+
+with open(scipp.config.config_path(), 'w') as config_file:
+    config_file.write('plot:\n')
+    config_file.write('  use_plopp: true\n')
```

Comment:
This overwrites the user's config!

Reply:
:innocent:

Reply:
@jl-wynen Found a different way, please have a look.

Unrelated: Need to figure out why the mechanism for config in working dir is not working.

Reply:
Looks good